lib/elixir_search_extractor_web/templates/shared/_app-navbar.html.eex

```diff
@@ -0,0 +1,5 @@
+<nav class="app-navbar">
+	<a href="/">
+		<img class="app-navbar__logo" src="<%= Routes.static_path(@conn, "/images/logo.svg") %>" alt="Logo"/>
+	</a>
```

Comment:
The indentation is quite different between layout file and the rest of template files. Let's update them to be consistent 💪

Reply:
@rosle Fixed in [ce2a0ae](https://github.com/rafayet-monon/elixir_search_extractor/pull/28/commits/ce2a0ae1e9a3da6731504d1d89d3576136bc3c1d). Also added localization which I forgot to add 🙈